cfc_app/tests.py

```diff
@@ -35,11 +35,34 @@ def test_health_status_is_up(self):
         self.assertContains(response, '{"status": "UP"}')
 
     def test_health_status_redirects(self):
-        """ Test that '/health' is redirected to '/health/' and returned with RC=301 """
+        """ Test that '/health' is redirected to '/health/' with RC=301 """
 
         response = self.client.get('/health')
-        self.assertEqual(response.status_code, 301)
-        self.assertEqual(response.url, '/health/')
+        self.assertRedirects(response, '/locations/', 301, 200)
+
+class LocationsEndpointTests(TestCase):
+    """ Locations Endpoint use to show all supported locations """
+
+    def test_locations_template(self):
+        """ Test that locations uses 'locations.html' template """
+
+        response = self.client.get('/locations/')
+        self.assertTemplateUsed(response, 'locations.html')
+
+    def test_locations_default_locations_loaded(self):
+        """ Test that default locations are loaded"""
```

Comment:
locations endpoint loads default location, not sure if this would be a robust test, any suggestions are welcome

Reply:
@jkevinruiz, I had a conversation with @tonypearson. This is a good test to check if the seed data was loaded successfully. However we don't think it should run every time code is committed to the repo. This is a data integrity test and not a unit test. We recommend moving this out of the unit test file and adding usage instructions in the README.md if somebody wanted to test loading the seed data.

Reply:
@upkarlidder that makes sense, i'll remove the default tests and update the readme. I'm guessing the other test works fine. 

Do you have any other test cases we could write around this endpoints? From examples I've seen endpoints are mostly tested with templates and the response data's they return in django.